### PR TITLE
Adds new section in README regarding development - Issue #107

### PR DESCRIPTION
**Pull request by gadamc**
This new README section includes step-by-step instructions to get started with development and some comments on adding new code to ensure additions and changes to the codebase go as smooth as possible.

* Adds new section to README called "Development"
* Addresses Issue #107 
* Added to README to increase visibility. Alternative would be to create a new document called "Development.md" and add information there. Open for discussion
* no testing required -- local Markdown renders as expected

### CODE REVIEW

**Comment by gadamc** [comment]
@NnguyenHTommy  Can you review this change for me?

File: README.md [hunk 1]
```diff
@@ -124,6 +124,95 @@ The console program `qt3piezo` comes installed via the 'nipiezojenapy' package,
...
+> pip install -e . 
+```
+
+5. Create a new branch for your work
+
+```
+> git checkout -b add-my-fancy-new-feature
+```
+
+6. Add your code and test, test, test!
```
Comment (NnguyenHTommy) [inline README.md:162]: Ideally would be to test on actual hardware

Comment (gadamc) [reply README.md:162]: I address this in the comments below, but I'll also add this here.

File: README.md [hunk 2]
```diff
@@ -124,6 +124,95 @@ The console program `qt3piezo` comes installed via the 'nipiezojenapy' package,
...
+
+7. Push your branch
+
+```
+> git push -u origin add-my-fancy-new-feature
+```
+
+8. Issue a pull request from this website
```
[the remark this answers is not in the document]
Comment (gadamc) [reply README.md:170]: I do mean qt3-utils. I'll be more explicit.

File: README.md [hunk 3]
```diff
@@ -124,6 +124,95 @@ The console program `qt3piezo` comes installed via the 'nipiezojenapy' package,
...
+```
+
+4. Install qt3-utils in "editor" mode
+
+```
+> cd qt3-utils
+> pip install -e . 
+```
+
+5. Create a new branch for your work
```
[the remark this answers is not in the document]
Comment (gadamc) [reply README.md:156]: Good question - I'll explain below. A new branch should be created for each new addition to the repo. I would discourage folks making a separate branch for your microscope. That will almost guarantee that branches get out of sync with the main branch. I think we should make it clear that if you are using qt3-utils for experimental work that doesn't involve updating qt3-utils, but you want to use git for version control of external scripts or jupyter notebooks, then you should create an entirely new git repository for that and `pip install qt3utils` in the normal way.

**Review by NnguyenHTommy — APPROVED** [review]
It looks good!

**Pull request merged**